src/pages/SupportPage.ts

```diff
@@ -0,0 +1,38 @@
+// import _ from 'lodash/fp';
+import { h, h2, p } from 'react-hyperscript-helpers';
+import FooterWrapper from 'src/components/FooterWrapper';
+import { PageBox } from 'src/components/PageBox';
+import TopBar from 'src/components/TopBar';
+import * as Nav from 'src/libs/nav';
+import * as Style from 'src/libs/style';
+import { SupportResourceList } from 'src/support/SupportResourceList';
+
+interface SupportPageProps {
+  queryParams: {
+    selectedType: string | undefined;
+    resourceName: string | undefined;
+  };
+}
+
+const SupportPage = (props: SupportPageProps) => {
+  const selectedType = props.queryParams.selectedType;
+  const resourceName = props.queryParams.resourceName;
+
+  return h(FooterWrapper, [
+    h(TopBar, { title: 'Support', href: Nav.getLink('support') }, []),
```

Comment:
I would suggest you write the file in TSX instead of using hyperscript helpers. We are working on transitioning away from hyperscript helpers.